### ISSUE TITLE
How I Accidentally Built a OpenTelemetry Simulator in NodeJS and what I learnt 

### ISSUE THREAD

**Issue by ahaw021** [BrisJS/meetups#466]
Talk title: How I Accidentally Built a OpenTelemetry Simulator in NodeJS and what I learnt 
Talk length: 30 min
Talk synopsis:

- My Christmas 2021 Project
- How NodeJS OpenTelemetry Libraries Helped
- Demo of what was produced (visualized) 
- Engineering Aspects of OpenTelemetry to consider
- Q&A 

Available month(s): Feb 2022 -> Onwards

**Comment by iamkevinv** [comment]
I love it @ahaw021 :) 
Can I confirm you for the next livestream on Monday, February 14, 2022!?

**Comment by ahaw021** [comment]
@iamkevinv  -- that works :D

**Comment by iamkevinv** [comment]
Hi @ahaw021 - i just realised that this BrisJS date of Feb 14 is valentines day. Hopefully that doesn't affect your availability.
Pretty sure you're andrei hawke right? (can't keep track of the github profiles without real names haha) Can you confirm... and also your preferred social network presence link for the meetup page and youtube page please? Otherwise i'd end up linking to this github profile for you.

**Comment by ahaw021** [comment]
hi @iamkevinv  -- ooppsss thanks for checking up and yes I have partner duties. Sorry to do this at the last minute but will need to do next month

**Comment by iamkevinv** [comment]
@ahaw021 - Can i confirm you for the 4th March Meetup? I'll get the writeup happening.

**Comment by iamkevinv** [comment]
Sorry - i meant 7th @ahaw021 :)

**Comment by ahaw021** [comment]
Hi Kevin. Book me in

**Comment by ahaw021** [comment]
Hey @iamkevinv  -- i am not feeling well today so won't be able to make it. 

Sorry for the late notice

**Comment by iamkevinv** [comment]
Oh no... sorry to hear that @ahaw021 shame we can't get a replacement in time now. Hope we can get you in later. Thanks

**Comment by iamkevinv** [comment]
Hi @ahaw021 - Not sure if you're available for this month? I'm trying to work out who will make this April lineup monday 11th. Can you let me know if you're keen for this one?

**Comment by ahaw021** [comment]
Hi @iamkevinv  - should be good. 

Though this won't be a 30 minute talk probably 10-15 as it's just more here's something different.

**Comment by iamkevinv** [comment]
Yeah mate, no worries, 15 mins is totally fine, we'll put Ashley on for a 10 min talk too, and we'll make it a double!

**Comment by ahaw021** [comment]
@iamkevinv  -- did a dry run this morning. About 17-20 minutes.

**Comment by iamkevinv** [comment]
Hi there! Please find the details to connect via zoom. Zoom does all the streaming to youtube for us. We'll connect 10 mins before and test out our connectivity, before muting and going live :) See you soon!

Topic: BrisJS Meetup
Time: Apr 11, 2022 06:30 PM Brisbane

Join Zoom Meeting
https://us02web.zoom.us/j/85153800585

Meeting ID: 851 5380 0585
One tap mobile
+61280156011,,85153800585# Australia
+61370182005,,85153800585# Australia

Dial by your location
        +61 2 8015 6011 Australia
        +61 3 7018 2005 Australia
        +61 7 3185 3730 Australia
        +61 8 6119 3900 Australia
        +61 8 7150 1149 Australia
Meeting ID: 851 5380 0585
Find your local number: https://us02web.zoom.us/u/kcM2vMYniv

**Comment by iamkevinv** [comment]
Thanks for presenting this talk! If you have resources to link for folks, please feel free to still attach them here!
